pkg/targetgroupbinding/networking_manager.go

```diff
@@ -225,7 +230,103 @@ func (m *defaultNetworkingManager) consolidateIngressPermissionsPerSGByTGB(_ con
 }
 
 // computeAggregatedIngressPermissionsPerSG will aggregate ingress permissions by SG across all TGBs.
-func (m *defaultNetworkingManager) computeAggregatedIngressPermissionsPerSG(_ context.Context) map[string][]networking.IPPermissionInfo {
+func (m *defaultNetworkingManager) computeAggregatedIngressPermissionsPerSG(ctx context.Context) map[string][]networking.IPPermissionInfo {
+	if m.disableRestrictedSGRules {
+		return m.computeUnrestrictedIngressPermissionsPerSG(ctx)
+	}
+	return m.computeRestrictedIngressPermissionsPerSG(ctx)
+}
+
+func (m *defaultNetworkingManager) groupIngressPermsBySourceAndProtocolPerSG(_ context.Context) (map[string][]networking.IPPermissionInfo, map[string]map[string]map[string][]networking.IPPermissionInfo) {
+	permsFromNLBPerSG := make(map[string][]networking.IPPermissionInfo)
```

Comment:
lets rename to permsFromIPRangeRulesPerSG or something similar

Reply:
Done.